crates/astria-core/src/protocol/transaction/v1alpha1/action.rs

```diff
@@ -787,6 +786,11 @@ pub struct Ics20Withdrawal {
     // if unset, and the transaction sender is a bridge account, the withdrawal is
     // treated as a bridge withdrawal (ie. the bridge account's withdrawer address is checked).
     pub bridge_address: Option<Address>,
+
+    // whether to use a bech32-compatible format of the `.return_address` when generating
+    // fungible token packets (as opposed to Astria-native bech32m addresses). This is
+    // necessary for chains like noble which enforce a strict bech32 format.
```

Comment:
why does the return address need to be bech32 compatible? if the withdrawal is returned, it'll be to astria. shouldn't this be for `destination_chain_address`?

Reply:
noble indeed checks the counterparty return address - unless I am mistaken about terminologies.

I have summarized the findings in this issue: https://github.com/astriaorg/astria/issues/1407

Do you see how I could improve the comment to make this clearer?